core/src/main/java/bisq/core/btc/setup/WalletsSetup.java

```diff
@@ -409,6 +420,64 @@ private void configPeerNodes(@Nullable Socks5Proxy proxy) {
         networkConfig.proposePeers(peers);
     }
 
+    ///////////////////////////////////////////////////////////////////////////////////////////
+    // Peer configuration check
+    ///////////////////////////////////////////////////////////////////////////////////////////
+
+    /* Provided a Peer and its VersionMessage, will check and handle Peer's advertised
+     * configuration. The VersionMessage is provided separately, because it is not
+     * always available and this is meant to be called from a listener that is triggered
+     * when the VersionMessage becomes available.
+     */
+    private void handleConfiguration(Peer peer, VersionMessage versionMessage) {
+        var wellConfigured = isWellConfigured(versionMessage);
+        if (wellConfigured) {
+            log.info("Peer well configured: {}", peer);
+        } else {
```

Comment:
Redundant `else` clause bc the `if` can return. Makes it more readable
```suggestion
        if (wellConfigured) {
            log.info("Peer well configured: {}", peer);
            return;
        }
```

Reply:
If you don't remove the `wellConfigured` intermediary variable as you suggested above (which I am against), this originally reads "if the node is well configured log that, else perform following steps", which is favourable over your suggetion: "if the node is well configured, given provided version message, exit the method, foregoing steps that would only be executed in case the node were not well configured." Your edits have a negative effect on readability and a very minimal effect on terseness, which, in my opinion, should not be a consideration anyway (but that's besides the point).